blocks/api/raw-handling/special-comment-converter.js

```diff
@@ -71,3 +70,16 @@ function createMore( customText, noTeaser ) {
 	}
 	return node;
 }
+
+function replace( processedNode, newNode ) {
```

Comment:
Minor: Worth adding these to dom utils? We're doing these in a few places I think, even outside the paste component. Could be done separately though.

Reply:
Done.

Reply:
To do: actually refactor elsewhere in the code base.